ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/match-lists.j2

```diff
@@ -0,0 +1,10 @@
+{# eos - match-lists #}
+{% if match_lists is arista.avd.defined %}
+!
+{%     for match_list in match_lists | arista.avd.natural_sort %}
+match-list input string {{ match_list }}
+{%         for sequence in match_lists[match_list].sequence_numbers | arista.avd.natural_sort %}
+   {{ sequence }} match {{ match_lists[match_list].sequence_numbers[sequence].match }}
```

Comment:
missing a test for `match_lists[match_list].sequence_numbers[sequence].match`

Reply:
Fixed in d045a8f9c6df50b2f372a593e770eae35a65b9d2